<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1532</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1351</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>271</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Tri et recherche :</string>
      </property>
      <widget class="QTableView" name="recherche_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>281</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter </string>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche_button">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_button">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>410</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche0">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>571</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>GESTION DES PARTENAIRES </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>440</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>     Pdf</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/PROJET QT/icones/recherche.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>430</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/PROJET QT/icones/pdf.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>130</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/PROJET QT/icones/supprimer.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>70</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/PROJET QT/icones/modifier.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>470</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/PROJET QT/icones/TRI.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/PROJET QT/icones/Ajout.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>430</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/placeholderblackshapeforlocalizationonmaps_79861.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>50</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher la liste des partenaires : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>490</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Excel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1391</width>
        <height>731</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Desktop/2EME/PROJET QT/d9ac02fa-15c8-40f1-becc-5bfe3902f3fb.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Contact :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="domaine_ajout">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>300</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresse_ajout">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Domaine :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contact_ajout">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>380</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="num_ajout">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="afficher">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>70</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supp_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>540</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer </string>
      </property>
     </widget>
     <widget class="QPushButton" name="bouton_modif">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>540</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="num_supp">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>500</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>470</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Saisir le numero de partenaire :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pr_ajout_">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_ajout">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="nom_ajout_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom_p :</string>
      </property>
     </widget>
     <widget class="QLabel" name="pr_ajout">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom_p :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>410</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Numero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>contact</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom_p</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>1230</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>localisation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>pushButton</zorder>
     <zorder>recherche_button</zorder>
     <zorder>tri_button</zorder>
     <zorder>recherche0</zorder>
     <zorder>label_6</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>label_12</zorder>
     <zorder>label_13</zorder>
     <zorder>label_14</zorder>
     <zorder>label_15</zorder>
     <zorder>label_16</zorder>
     <zorder>label_17</zorder>
     <zorder>label_20</zorder>
     <zorder>label_5</zorder>
     <zorder>pushButton_7</zorder>
     <zorder>label_3</zorder>
     <zorder>domaine_ajout</zorder>
     <zorder>adresse_ajout</zorder>
     <zorder>label_4</zorder>
     <zorder>label</zorder>
     <zorder>contact_ajout</zorder>
     <zorder>label_2</zorder>
     <zorder>num_ajout</zorder>
     <zorder>afficher</zorder>
     <zorder>supp_button</zorder>
     <zorder>bouton_modif</zorder>
     <zorder>num_supp</zorder>
     <zorder>label_8</zorder>
     <zorder>pr_ajout_</zorder>
     <zorder>nom_ajout</zorder>
     <zorder>nom_ajout_2</zorder>
     <zorder>pr_ajout</zorder>
     <zorder>comboBox_2</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_3</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1532</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
